lib/envars.ml

```diff
@@ -205,6 +205,7 @@ let print_config ?(prefix_var_name="") f coq_src_subdirs =
   let open Printf in
   fprintf f "%sLOCAL=%s\n" prefix_var_name (if Coq_config.local then "1" else "0");
   fprintf f "%sCOQLIB=%s/\n" prefix_var_name (coqlib ());
+  fprintf f "%sCOQCORELIB=%s/\n" prefix_var_name (if Coq_config.local then coqlib () else coqlib () / "../coq-core/");
```

Comment:
This is fishy: when the build is not local but the test-suite is run `coqlib ()` returns the path of the source. In this case, the sibling directory `coq-core` either does not exist or is unrelated.

Reply:
Umm, at least in the regular CI `coqlib` will return the installed path so indeed while hacky this should work, how do you call the test suite to get coqlib to point to the source in non-local build?

Reply:
Problem identified but was due to incorrect setting of COQLIB in coq makefile, actually the OSX job did also incorrectly set OCAMLPATH.